charts/home-assistant/README.md

```diff
@@ -1,189 +1,114 @@
-# Home Assistant
+# home-assistant
```

Comment:
We should maybe have a guideline for what we put here... is it the chart name, the app name? If it's the latter I think we should stick with `Home Assistant`

Reply:
"Home Assistant" is good with me too, we should be using app names